.github/workflows/test-spras.yml

```diff
@@ -61,141 +61,6 @@ jobs:
       shell: bash --login {0}
       run: snakemake --cores 2 --configfile config/config.yaml --show-failed-logs
 
-  # Builds the Docker images
-  docker:
-    name: Build Docker images
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        os: [ubuntu-latest]
-    steps:
-    - name: Checkout repository
-      uses: actions/checkout@v2
-      # Pull from Docker Hub to use the cache
-      # https://medium.com/mobileforgood/coding-tips-patterns-for-continuous-integration-with-docker-on-travis-ci-9cedb8348a62
-      # https://github.com/docker/build-push-action/issues/7
-    - name: Pull Docker images
-      run: |
-        docker pull reedcompbio/omics-integrator-1:latest
-        docker pull reedcompbio/omics-integrator-2:v2
-        docker pull reedcompbio/pathlinker:v2
-        docker pull reedcompbio/meo:latest
-        docker pull reedcompbio/mincostflow:latest
-        docker pull reedcompbio/allpairs:v2
-        docker pull reedcompbio/domino:latest
-        docker pull reedcompbio/py4cytoscape:v3
-        docker pull reedcompbio/spras:v0.1.0
-    - name: Build Omics Integrator 1 Docker image
-      uses: docker/build-push-action@v1
-      with:
-        path: docker-wrappers/OmicsIntegrator1/.
-        dockerfile: docker-wrappers/OmicsIntegrator1/Dockerfile
-        repository: reedcompbio/omics-integrator-1
-        tags: latest
-        cache_froms: reedcompbio/omics-integrator-1:latest
-        push: false
-    - name: Remove Omics Integrator 1 Docker image
-      # Remove the image to prevent the cache from being used. Here we use
-      # `|| true` to prevent the job from failing if the image doesn't exist or
-      # can't be removed for some reason
```

Comment:
Could transfer this comment to the new `docker rmi`

Reply:
Addressed in a new commit.